02-express-tutorial/routes/people.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express')
```

Comment:
I think the instructor moves this file to `13-router-people.js` inside the `02-express-tutorial` folder so you can do that too if you want the numbers to line up, but it's probably fine either way.

Reply:
I think I will leave the last three files in the folders to follow the idea to divide the controllers and routes.